demos/browser/server.js

```diff
@@ -1,153 +1,105 @@
-      }
-      const joinInfo = {
-        JoinInfo: {
-          Title: title,
-          Meeting: meetingCache[title].Meeting,
-          Attendee: (
-            await chime
-              .createAttendee({
-                MeetingId: meetingCache[title].Meeting.MeetingId,
-                ExternalUserId: uuid(),
-              })
-              .promise()
-          ).Attendee,
-        },
-      };
-      attendeeCache[title][joinInfo.JoinInfo.Attendee.AttendeeId] = name;
-      response.statusCode = 201;
-      response.setHeader('Content-Type', 'application/json');
-      response.write(JSON.stringify(joinInfo), 'utf8');
-      response.end();
-      log(JSON.stringify(joinInfo, null, 2));
-    } else if (request.method === 'GET' && request.url.startsWith('/attendee?')) {
-      const query = url.parse(request.url, true).query;
-      const attendeeInfo = {
-        AttendeeInfo: {
-          AttendeeId: query.attendee,
-          Name: attendeeCache[query.title][query.attendee],
-        },
-      };
-      response.statusCode = 200;
-      response.setHeader('Content-Type', 'application/json');
-      response.write(JSON.stringify(attendeeInfo), 'utf8');
-      response.end();
-      log(JSON.stringify(attendeeInfo, null, 2));
-    } else if (request.method === 'POST' && request.url.startsWith('/meeting?')) {
-      const query = url.parse(request.url, true).query;
-      const title = query.title;
-      if (!meetingCache[title]) {
-        meetingCache[title] = await chime
-          .createMeeting({
-            ClientRequestToken: uuid(),
-            // NotificationsConfiguration: {
-            //   SqsQueueArn: 'Paste your arn here',
-            //   SnsTopicArn: 'Paste your arn here'
-            // }
-          })
-          .promise();
-        attendeeCache[title] = {};
+    // Enable HTTP compression
+    compression({})(request, response, () => {});
+    const requestUrl = url.parse(request.url, true);
+    if (request.method === 'GET' && requestUrl.pathname === '/') {
+      // Return the contents of the index page
+      respond(response, 200, 'text/html', indexPage);
+    } else if (request.method === 'POST' && requestUrl.pathname === '/join') {
+      // Look up the meeting by its title. If it does not exist, create the meeting.
+      if (!meetingTable[requestUrl.query.title]) {
+        meetingTable[requestUrl.query.title] = await chime.createMeeting({
+          // Use a UUID for the client request token to ensure that any request retries
+          // do not create multiple meetings.
+          ClientRequestToken: uuid(),
+          // Specify the media region (where the meeting is hosted).
+          // In this case, we use the region selected by the user.
+          MediaRegion: requestUrl.query.region,
+          // Any meeting ID you wish to associate with the meeting.
+          // For simplicity here, we use the meeting title.
+          ExternalMeetingId: requestUrl.query.title,
```

Comment:
`requestUrl.query.title.substring(0, 64);` to avoid 400 in case the title is too long.

Reply:
Fixed